src/Object.cpp

```diff
@@ -18,9 +18,7 @@
 namespace osgviz {
 
 Object::Object():cull_mask(0xffffffff),visible(true){
-    text_size = 0.1;
-    text_position = osg::Vec3(0,0,0);
-    text_color = osg::Vec4(0,0,0,1);
+
     text = NULL;
```

Comment:
maybe move `text = NULL` to the initializer list?

Reply:
as it is a osg::ref_ptr, it doesn't need to be initialized at all...